### PR TITLE
Optimize the generate command for better execution time

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose

Fixes:[4006](https://github.com/ballerina-platform/ballerina-standard-library/issues/4006)

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java [hunk 1]
```diff
@@ -41,6 +41,8 @@ private PersistToolsConstants() {}
     public static final String KEY_PASSWORD = "password";
     public static final String KEY_DATABASE = "database";
     public static final String KEY_HOST = "host";
+    public static final String KEYWORD_PACKAGE = "package";
+    public static final String KEYWORD_NAME = "name";
```
Comment (niveathika) [inline persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java:45]: There should be constants for this in ballerina lang. Lets use that.

**Review by niveathika — COMMENTED** [review]
Lets check the same flow in init command too. Here if the directory is emoty we have to create file with same name as the package isnt it? We can improve that too.

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java [hunk 2]
```diff
@@ -130,6 +134,35 @@ public static PersistConfiguration readPersistConfigurations(String schemaName,
...
+                            if (field.identifier().toSourceCode().trim()
+                                    .equals(KEYWORD_NAME)) {
+                                return field.value().toSourceCode().trim().replaceAll("\"", "");
+                            }
+                        }
+                    }
+
+                }
+            }
+            throw new BalException("Error while reading Ballerina.toml. Package name could not be read. ");
```
Comment (daneshk) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java:160]:
```suggestion
            throw new BalException("Couldn't find package name in the Ballerina.toml file.");
```

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java [hunk 3]
```diff
@@ -130,6 +134,35 @@ public static PersistConfiguration readPersistConfigurations(String schemaName,
...
+        } catch (IOException e) {
+            throw new BalException("error while Ballerina.toml. " + e.getMessage());
```
Comment (daneshk) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java:162]:
```suggestion
            throw new BalException("Couldn't read the Ballerina.toml file. " + e.getMessage());
```

File: persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java (hunk 1 above)
Comment (sahanHe) [reply persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java:45]: > Lets check the same flow in init command too. Here if the directory is emoty we have to create file with same name as the package isnt it? We can improve that too.
this was updated in [5b06764](https://github.com/ballerina-platform/persist-tools/pull/127/commits/5b06764e6b339d0cd379fd6a908b8c2ec32a7107). For the constant from lang side, I couldn't find it in the `io.ballerina.runtime.api.constants` and `org.wso2.ballerinalang.compiler.util` packages. Is there any particular place to look?

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java (hunk 3 above)
Comment (sahanHe) [reply persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java]: updated in [3d46762](https://github.com/ballerina-platform/persist-tools/pull/127/commits/3d467623d38eb6d6ce04ffb0efbaf105f1dafe7f)

**Comment by daneshk** [comment]
What is the execution time of init command with this fix.

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java [hunk 4]
```diff
@@ -130,6 +134,35 @@ public static PersistConfiguration readPersistConfigurations(String schemaName,
...
+                            if (field.identifier().toSourceCode().trim()
+                                    .equals(KEYWORD_NAME)) {
+                                return field.value().toSourceCode().trim().replaceAll("\"", "");
+                            }
+                        }
+                    }
+
+                }
+            }
+            throw new BalException("Couldn't find package name in the Ballerina.toml file.");
```
Comment (daneshk) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java:160]:
```suggestion
            throw new BalException("couldn't find package name in the Ballerina.toml file.");
```

File: persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java [hunk 5]
```diff
@@ -130,6 +134,35 @@ public static PersistConfiguration readPersistConfigurations(String schemaName,
...
+        } catch (IOException e) {
+            throw new BalException("Couldn't read the Ballerina.toml file. " + e.getMessage());
```
Comment (daneshk) [inline persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java:162]:
```suggestion
            throw new BalException("couldn't read the Ballerina.toml file. " + e.getMessage());
```

**Review by daneshk — APPROVED** [review]
LGTM

File: persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java (hunk 1 above)
Comment (niveathika) [reply persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java:45]: Have you checked in PackageConstants?

**Review by daneshk — APPROVED** [review]

File: persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java (hunk 1 above)
Comment (sahanHe) [reply persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java:45]: This change will be done in a later update.

**Pull request merged**